File: MyTestProject/Site.WEB/src/app/_models/transaction.ts
export class Transaction {

    sender: string;
    receiver: string;
    password: string;
    amount: number;
    confirmed: boolean;

    constructor(sender?: string, receiver?: string, password?: string, amount?: number, confirmed?: boolean) {
        this.sender = sender;
        this.receiver = receiver;
        this.password = password;
        this.amount = amount;
        this.confirmed = confirmed;
    }
}

File: MyTestProject/Site.Models/Entities/Transaction.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Site.Models.Entities
{
  public class Transaction : Entity<Guid>
  {
    public Guid UserId { get; set; }

    public string Sender { get; set; }

    public string Receiver { get; set; }

    public decimal Amount { get; set; }

    public bool Confirmed { get; set; }

  }
}
